src/Models/SongsModel.php

```diff
@@ -39,4 +39,12 @@ public function getSongsAndAlbumByAlbum(int $id): array
         $query->execute(['id' => $id]);
         return $query->fetchAll();
     }
+
+    public function updateDatabase(int $songId): bool
```

Comment:
Ok we should rename this as well, updateDatabase is way to generic

Reply:
updatePlayCount